include/eve/std.hpp

```diff
@@ -0,0 +1,32 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/wide.hpp>
+#include <eve/logical.hpp>
+
+namespace eve::experimental
+{
+  template<typename T>
+  using simd = eve::wide<T, eve::fixed<16/sizeof(T)>>;
```

Comment:
Rly? Always 16?

Reply:
I dunno, clang and g++ gives different size. I purposedly let this and I await the failure to update accordingly.

Reply:
OK. Let's do that.